assets/aaVideo.qml

```diff
@@ -0,0 +1,8 @@
+import QtQuick 2.0

```

Comment:
can we follow the naming convention for this? so `aa_video.qml` instead? And then just update it in the resources file and in the code

Never done qml before, is the width/height just a default value?

Reply:
We *should* be able to assign width and height directly to variables. But every time I tried this it didn't work, or worse, segfaulted for some reason. But was happy with default values that are immediately overwritten.